schema/tables/login_password.yml

```diff
@@ -0,0 +1,12 @@
+name: login_password
+platforms:
+  - macOS
+description: Options of login and password (e.g password hints enabled).
+columns:
+  - name: password_hint_enabled
+    type: integer
+    required: false
+    description: whether password hint is enabled. 1 is enabled. 0 is not enabled.
```

Comment:
`whether password hint is enabled for any user. 1 means one or more users has a password hint set, 0 means no user has a password hint set.`